tests/templates/kuttl/kerberos/21-unleash-the-chaosmonkey.yaml.j2

```diff
@@ -0,0 +1,49 @@
+# Tribute to https://github.com/Netflix/chaosmonkey
+
+# We need to force-delete the Pods, because IONOS is sometimes unable to delete the pod (it's stuck in Terminating for > 20 minutes)
```

Comment:
That's worrying... do we have access to IONOS' kubelet logs? Is this an IONOS-only issue?

Reply:
Update: Support recommended using https://github.com/kvaps/kubectl-node-shell, which worked.
Sadly it only logs stuff during startup and than remains silent.
```
journalctl -u kubelet --since "48 hour ago"
-- Logs begin at Sat 2023-06-03 19:51:35 UTC, end at Tue 2023-06-06 12:01:05 UTC. --
-- No entries --
```
Anyway, even with force deletion tests have random timeouts on IONOS after we added the chaosmonkey. Everything is really slow